pkg/config/config_test.go

```diff
@@ -23,9 +24,12 @@ func setupConf() Config {
 
 func setupConfFromYAML(yamlConfig string) Config {
 	conf := NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
-	conf.SetConfigType("yaml")
-	conf.ReadConfig(bytes.NewBuffer([]byte(yamlConfig)))
 	initConfig(conf)
+	conf.SetConfigType("yaml")
+	e := conf.ReadConfig(bytes.NewBuffer([]byte(yamlConfig)))
+	if e != nil {
+		log.Println(e)
```

Comment:
Shouldn't this be fatal and fail the test?

Reply:
og.Fatal would abort all the tests. I could assert(false) or something like that (after printing the error anyway).